src/UrlGenerator/UrlGenerator.php

```diff
@@ -16,6 +17,16 @@
     public function getUrl(): string;
 
     /**
+     * Get the temporary url for the profile of a media item.
+     *
+     * @param \DateTimeInterface $expiration
+     * @param array              $options
+     *
+     * @return string
+     */
+    public function getTemporaryUrl(DateTimeInterface $expiration, array $options = []): string;
```

Comment:
@AlexVanderbist Did not spot this when reviewing, but adding a method to an interface is a breaking change (see #765).

Could you tag a new bugfix release where this method is removed from the interface. Also add a new issue with a label `revisit for next major version` with a note that we should re-add that method to the interface.